### PR TITLE
feat(ie fixes): removed types that are unsupported in IE11

### PR DESCRIPTION
**Pull request by m-suchorski**
IE fixes for #102

### CODE REVIEW

File: src/util/polyfills.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+
+Array.prototype.find = Array.prototype.find || function (callback) {
```
Comment (mateuszluczak) [inline src/util/polyfills.js]: Can't we just replace find with filter where it's used?
Is this so critical to have a polyfill for that?

File: src/core/registry.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ class Registry {
         this._isRegistrationScheduled = true;
 
         window.requestAnimationFrame(() => {
-          const ev = new Event('content:loaded');
+          const ev = createCustomEvent('content:loaded');
```
Comment (mateuszluczak) [inline src/core/registry.js]: Can't you use Element API here?
`$(window).trigger('content:loaded')`

File: src/util/polyfills.js (hunk 1 above)
Comment (m-suchorski) [reply src/util/polyfills.js]: Makes sense, gonna try that.

File: src/core/registry.js (hunk 2 above)
Comment (m-suchorski) [reply src/core/registry.js]: Right, it might work. I'll try that.

File: src/util/polyfills.js (hunk 1 above)
Comment (m-suchorski) [reply src/util/polyfills.js]: Done.

File: src/core/registry.js (hunk 2 above)
Comment (m-suchorski) [reply src/core/registry.js]: Done.

**Pull request merged**